tapalcatl_server/handler.go

```diff
@@ -468,7 +471,14 @@ func MetatileHandler(p Parser, metatileSize int, mimeMap map[string]string, stor
 		reqState.Coord = &parseResult.Coord
 		reqState.HttpData = &parseResult.HttpData
 
-		metaCoord, offset := parseResult.Coord.MetaAndOffset(metatileSize)
+		metaCoord, offset, err := parseResult.Coord.MetaAndOffset(metatileSize, tileSize)
+		if err != nil {
+			configErrors.Add(1)
+			logger.Warning(LogCategory_ConfigError, "MetaAndOffset could not be calculated: %s", err.Error())
+			http.Error(rw, err.Error(), http.StatusInternalServerError)
+			reqState.ResponseState = ResponseState_Error
+			return
```

Comment:
Well spotted, thanks!

Should it be `FetchState_ConfigErr` or `FetchState_Nil`? At this point in the code, a fetch hasn't happened yet, so `FetchState_ConfigErr` is slighly confusing to me - it seems to be saying that the fetch failed because of a config error.